### PR TITLE
Results Title Ticket #56 and fixed poppins font

### PR DESCRIPTION
**Pull request by Raymond0T**
#### What is the purpose of your change?
-Creating Results Title for ticket #56 and imported poppins font 

#### What areas of the application does this change impact? (changes API, folder structure, etc)
-results page and font for the app

#### What features/fixes are deferred/not handled in this PR?
-Spacing and font sizing

#### Screenshot or link to preview
-https://deploy-preview-92--pick-a-major.netlify.app/guide
image
## Checklist
- [ ] Adds unit tests
- [ ] Has a work item linked to the PR

### CODE REVIEW

File: tmt-app/public/index.html [hunk 1]
```diff
@@ -11,7 +15,7 @@
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <!--
-      manifest.json provides metadata used when your web app is installed on a
+      manifest.json provides metadata used when your web application is installed on a
```
Comment (jwu910) [inline tmt-app/public/index.html:18]: What was the need for this change?

File: tmt-app/public/index.html [hunk 2]
```diff
@@ -3,6 +3,10 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
```
Comment (jwu910) [inline tmt-app/public/index.html:8]: FYI there is an npm package for the popping font, we can ship it as a dependency rather than a cdn

**Comment by jwu910** [comment]
Do you have a link to preview the code here? Or screenshots?

File: tmt-app/public/index.html (hunk 1 above)
Comment (Raymond0T) [reply tmt-app/public/index.html:18]: Nothing. I was trying to change the LF for the file but git does not register the change

File: tmt-app/public/index.html (hunk 2 above)
Comment (Raymond0T) [reply tmt-app/public/index.html:8]: Is there a specific npm package or would this one be fine [https://www.npmjs.com/package/@fontsource/poppins](url)

File: tmt-app/public/index.html (hunk 1 above)
Comment (jwu910) [reply tmt-app/public/index.html:18]: Can we leave the out of scope changes out?

Comment (jwu910) [reply tmt-app/public/index.html:18]: Also git should auto conver crlf(?) to LF if working on windows

File: tmt-app/public/index.html (hunk 2 above)
Comment (jwu910) [reply tmt-app/public/index.html:8]: I believe this one should be fined

**Review by jwu910 — APPROVED** [review]

**Pull request merged**